webextension/js/popup.js

```diff
@@ -159,6 +163,15 @@ const Logic = {
     }).catch((e) => {throw e;});
   },
 
+  getPanelSelector(panel) {
```

Comment:
This is fine, I perhaps would have used a translation based approach where the strings are embedded in some JSON and some method looks up the variation translation, this would potentially be a little less coupled to having this method.